webui/public/index.html

```diff
@@ -31,20 +31,23 @@
     <script src="%PUBLIC_URL%/js/pdf/build/pdf.js"></script>
     
     <script src="%PUBLIC_URL%/js/keycloak.js"></script>
-
+    <script src="%PUBLIC_URL%/js/arrive.min.js"></script>
     <script src="%PUBLIC_URL%/js/jquery-3.3.1.slim.min.js"></script>
     <script src="%PUBLIC_URL%/js/popper.min.js"></script>
     <script src="%PUBLIC_URL%/js/bootstrap.min.js"></script>
     <script src="%PUBLIC_URL%/js/Chart.bundle.js"></script>
     <script src="%PUBLIC_URL%/js/material.min.js"></script>
+    <script src="%PUBLIC_URL%/js/vega.min.js"></script>
```

Comment:
Will this work with only vega-lite or do we need full vega and vega embed as well? Mostly asking to reduce page size, though I know the vega lite specs are much simpler to use if they have the charts we need.

Reply:
I tested and confirmed we need all 3 of these.